LibreNMS/Validations/Php.php

```diff
@@ -59,7 +59,7 @@ private function checkVersion(Validator$validator)
     private function checkSessionDirWritable(Validator $validator)
     {
         $path = session_save_path() === '' ? '/tmp' : session_save_path();
-        if (!is_writable($path)) {
+        if (!is_writable($path) and ini_get(save_handler) == "file") {
```

Comment:
We generally use && instead of `and` and put quotes around save_handler, it is a string, not a constant.